### PR TITLE
make landing page pog af

### PR DESCRIPTION
**Pull request by zax-xyz**

### CODE REVIEW

**Review by zax-xyz — COMMENTED** [review]
epic

File: pages/index.tsx [hunk 1]
```diff
@@ -14,36 +15,78 @@ const DynamicBasedCountdown = dynamic(
...
-        date={new Date(2022, 9, 28, 15)}
-        completed={completed}
-        setCompleted={setCompleted}
-      />
-    </LandingLayout>
+
+        {/* <Card className="top-8 left-4" /> */}
+      </LandingLayout>
+
+      <Thingy />
```
Comment (maxphillipsdev) [inline pages/index.tsx:88]: @zaxutic Beautiful naming lmao

File: styles/globals.css [hunk 2]
```diff
@@ -8,7 +8,7 @@
 /* Import Nevis from public folder */
 @font-face {
   font-family: "Nevis";
-  src: url("/fonts/Nevis.ttf");
+  src: url("/fonts/nevis.ttf");
```
Comment (maxphillipsdev) [inline styles/globals.css:11]: bruh

**Review by maxphillipsdev — APPROVED** [review]
based and PR pilled

File: styles/globals.css (hunk 2 above)
Comment (zax-xyz) [reply styles/globals.css:11]: filesystem case sensitivity moment

**Pull request merged**